components/team_members_modal/team_members_modal.tsx

```diff
@@ -14,17 +13,26 @@ import InvitationModal from 'components/invitation_modal';
 
 import {ModalIdentifiers} from 'utils/constants';
 
-export default class TeamMembersModal extends React.PureComponent {
-    static propTypes = {
-        currentTeam: PropTypes.object.isRequired,
-        onHide: PropTypes.func.isRequired,
-        onLoad: PropTypes.func,
-        actions: PropTypes.shape({
-            openModal: PropTypes.func.isRequired,
-        })
-    }
+import {openModal} from 'actions/views/modals';
+
+type Props = {
+    currentTeam: {
+        id: string;
+        display_name: string;
+    };
+    onHide: () => void;
+    onLoad?: () => void;
+    actions?: {
```

Comment:
I don't think actions needs to be optional here. That'd also let us remove the null check you added below as well.

Reply:
I made it optional because the original definition did not have it required, but I can make it required.

Reply:
Thanks! I assume it was just missed when we added it originally